packages/server/modules/workspaces/tests/integration/repositories.spec.ts

```diff
@@ -140,7 +140,13 @@ describe('Workspace repositories', () => {
       const { id: userId } = await createAndStoreTestUser()
       const { id: workspaceId } = await createAndStoreTestWorkspace()
 
-      await upsertWorkspaceRole({ userId, workspaceId, role: 'workspace:member' })
+      await upsertWorkspaceRole({
+        userId,
+        workspaceId,
+        role: 'workspace:member',
+        createdAt: new Date(),
```

Comment:
seems like there's a lot of duplication here, each of those upsertWorkspaceRole calls has only 1-2 different lines. that said - up to you to evaluate if they are fundamentally duplicate or only temporarily duplicate and probably gonna change a lot from each other in the future